math/exp.c

```diff
@@ -22,9 +22,13 @@
 double frexp(double x, int *exp)
 {
 	conv_t *conv = (conv_t *)&x;
-
 	*exp = 0;
 
+	if (isnan(x) != 0) {
+		errno = EDOM;
```

Comment:
Does any standard/specification dictate that `errno` should be set?

Applies to other functions.

Reply:
Good catch, neither IEC 60559 nor ISO/IEC 9899:1999 mandates it.